packages/tools/examples/PolySegWasmContourToVolumeLabelmap/index.ts

```diff
@@ -0,0 +1,284 @@
+const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
+const segmentationId = 'MY_SEGMENTATION_ID';
+
+// ======== Set up page ======== //
+
+const size = '500px';
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.display = 'flex';
+viewportGrid.style.display = 'flex';
+viewportGrid.style.flexDirection = 'row';
+
+const element1 = document.createElement('div');
+const element2 = document.createElement('div');
+element1.style.width = size;
+element1.style.height = size;
+element2.style.width = size;
+element2.style.height = size;
+
+// Disable right click context menu so we can have right click tools
+element1.oncontextmenu = (e) => e.preventDefault();
+element2.oncontextmenu = (e) => e.preventDefault();
+
+viewportGrid.appendChild(element1);
+viewportGrid.appendChild(element2);
+
+content.appendChild(viewportGrid);
+
+createInfoSection(content, { ordered: true })
+  .addInstruction('Draw a contour segmentation on the left viewport')
+  .addInstruction(
+    'Click on the button to convert the contour segmentation to a volume labelmap segmentation'
+  );
+
+// ============================= //
+let toolGroup1, toolGroup2;
+let renderingEngine;
+const toolGroupId1 = 'ToolGroup_Contour';
+const toolGroupId2 = 'ToolGroup_Labelmap';
+const viewportId1 = 'CT_SAGITTAL_CONTOUR';
+const viewportId2 = 'CT_SAGITTAL_LABELMAP';
+
+const segmentIndexes = [1, 2, 3, 4, 5];
+
+addButtonToToolbar({
+  title: 'Convert contour segmentation to labelmap segmentation',
+  onClick: async () => {
+    // add the 3d representation to the 3d toolgroup
+    await segmentation.addSegmentationRepresentations(toolGroupId2, [
+      {
+        segmentationId,
+        type: csToolsEnums.SegmentationRepresentations.Labelmap,
+        options: {
+          polySeg: true,
+        },
+      },
+    ]);
+  },
+});
+
+addDropdownToToolbar({
+  labelText: 'Segment Index',
+  options: { values: segmentIndexes, defaultValue: segmentIndexes[0] },
+  onSelectedValueChange: (number) => {
+    segmentation.segmentIndex.setActiveSegmentIndex(
+      segmentationId,
+      Number(number) as number
+    );
+  },
+});
+
+/**
+ * Runs the demo
+ */
+async function run() {
+  // Init Cornerstone and related libraries
+  await initDemo();
+
+  // Add tools to Cornerstone3D
+  cornerstoneTools.addTool(PanTool);
+  cornerstoneTools.addTool(ZoomTool);
+  cornerstoneTools.addTool(StackScrollMouseWheelTool);
+  cornerstoneTools.addTool(SegmentationDisplayTool);
+  cornerstoneTools.addTool(PlanarFreehandContourSegmentationTool);
+  cornerstoneTools.addTool(SegmentSelectTool);
+
+  // Define tool groups to add the segmentation display tool to
+  toolGroup1 = ToolGroupManager.createToolGroup(toolGroupId1);
+  toolGroup2 = ToolGroupManager.createToolGroup(toolGroupId2);
+
+  // Manipulation Tools
+  toolGroup1.addTool(PanTool.toolName);
```

Comment:
Same comment as for surface - add the interpolation tools so it is easy to create a volume.